task/06-conditions-n-loops-tasks.js

```diff
@@ -356,7 +502,14 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 10 => '365'
  */
 function toNaryString(num, n) {
-    throw new Error('Not implemented');
+     
+    let res = '';
+    while(num >= n){
+        res = num % n + res;
+        num = Math.floor(num/n);
+        
+    }
+    return num + res;
```

Comment:
Please, try to use toString() method

https://www.w3schools.com/jsref/jsref_tostring_number.asp